From: Gustavo Noronha Silva <kov@debian.org>
Date: Tue, 17 Jul 2018 16:56:50 +0300
Subject: _tcdrain_kfreebsd

---
 libgksu/libgksu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libgksu/libgksu.c b/libgksu/libgksu.c
index a764ac2..9a0478f 100644
--- a/libgksu/libgksu.c
+++ b/libgksu/libgksu.c
@@ -2202,7 +2202,9 @@ gksu_su_fuller (GksuContext *context,
 	      write (fdpty, line, strlen(line));
 	      g_free (line);
 
+#ifndef __FreeBSD_kernel__
 	      tcdrain (fdpty);
+#endif
 
 	      bzero (buf, 256);
 	      read (fdpty, buf, 255);
