From: Gustavo Noronha Silva <kov@debian.org>
Date: Tue, 17 Jul 2018 16:56:50 +0300
Subject: _format-security

===================================================================
---
 libgksu/libgksu.c | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/libgksu/libgksu.c b/libgksu/libgksu.c
index 9a0478f..de91a3a 100644
--- a/libgksu/libgksu.c
+++ b/libgksu/libgksu.c
@@ -1276,7 +1276,7 @@ sudo_prepare_xauth (GksuContext *context)
   context->dir = g_strdup (mkdtemp(template));
   if (!context->dir)
     {
-      fprintf (stderr, strerror(errno));
+      fprintf (stderr, "%s", strerror(errno));
       return FALSE;
     }
 
@@ -2247,7 +2247,7 @@ gksu_su_fuller (GksuContext *context,
 					  " %s"), converted_str, "gksu: waiting");
 	      g_free (converted_str);
 
-	      g_set_error (error, gksu_quark, GKSU_ERROR_HELPER, emsg);
+	      g_set_error_literal (error, gksu_quark, GKSU_ERROR_HELPER, emsg);
 	      g_free (emsg);
 
 	      if (context->debug)
@@ -2979,8 +2979,11 @@ gksu_sudo_fuller (GksuContext *context,
 	}
     }
 
-  fprintf(stderr, child_stderr);
-  g_free(child_stderr);
+  if (child_stderr != NULL)
+    {
+      fprintf(stderr, "%s", child_stderr);
+      g_free(child_stderr);
+    }
 
   /* if error is set we have found an error condition */
   return (error == NULL);
