From: "Hideki Yamane (Debian-JP)" <henrich@debian.or.jp>
Date: Tue, 17 Jul 2018 16:56:50 +0300
Subject: Enable Japanese translations

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index f3d3a4f..b89a71c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -44,7 +44,7 @@ PKG_CHECK_MODULES(LIBGKSU, [gtk+-2.0 >= 2.4.0, gconf-2.0, libstartup-notificatio
 PKG_CHECK_MODULES(GKSU_PROPERTIES, [gtk+-2.0 >= 2.12, gconf-2.0])
 
 # Checks for library functions.
-ALL_LINGUAS="ca cs da de es eu fr hu it ko lt pl pt_BR ro ru sk sv nb nl th zh_CN"
+ALL_LINGUAS="ca cs da de es eu fr hu it ja ko lt pl pt_BR ro ru sk sv nb nl th zh_CN"
 
 AC_DEFINE(GETTEXT_PACKAGE, AC_PACKAGE_NAME, [gettext package])
 GETTEXT_PACKAGE=AC_PACKAGE_NAME
