From: Gustavo Noronha Silva <kov@debian.org>
Date: Tue, 17 Jul 2018 16:56:51 +0300
Subject: _unguarded_child_stderr

---
 libgksu/libgksu.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libgksu/libgksu.c b/libgksu/libgksu.c
index de91a3a..3872273 100644
--- a/libgksu/libgksu.c
+++ b/libgksu/libgksu.c
@@ -2951,7 +2951,8 @@ gksu_sudo_fuller (GksuContext *context,
 
       if (WEXITSTATUS(status))
 	{
-          if (g_str_has_prefix(child_stderr, "Sorry, user "))
+          if (child_stderr != NULL && 
+              g_str_has_prefix(child_stderr, "Sorry, user "))
             {
               g_set_error (error, gksu_quark, GKSU_ERROR_NOT_ALLOWED,
                            _("The underlying authorization mechanism (sudo) "
