From: Gustavo Noronha Silva <kov@debian.org>
Date: Tue, 17 Jul 2018 16:56:51 +0300
Subject: _sudo_wrong_pwd_check

---
 libgksu/libgksu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgksu/libgksu.c b/libgksu/libgksu.c
index 44dce5e..56edae8 100644
--- a/libgksu/libgksu.c
+++ b/libgksu/libgksu.c
@@ -2899,7 +2899,7 @@ gksu_sudo_fuller (GksuContext *context,
 	  fprintf (stderr, "%s", buffer);
 	}
 
-      if (g_str_has_prefix (buffer, "Sorry, try again."))
+      if (g_str_has_prefix (buffer, "Sorry, try again.") || g_str_has_prefix (buffer, "GNOME_SUDO_PASSSorry, try again."))
 	g_set_error (error, gksu_quark, GKSU_ERROR_WRONGPASS,
 		     _("Wrong password."));
       else
